### ISSUE TITLE
Scraped Data does not include anything more recent than 2013

### ISSUE THREAD

**Issue by samjett247** [stev-ou/review_ocr#2]
For the new instructor chip we built, we need to know how long the prof. has been at OU. But given the current data, we can only know 6 years back in the scraped dataset. Would this be a relatively easy fix to scrape all of the data @jlovoi ?

**Comment by jlovoi** [comment]
Yeah, I can go ahead a scrape all the data for sure, just hoping that there isn’t anything kooky and wacky and crazy about those PDFs.... I guess it wouldn’t be an issue anyway since it would only need to be checking for a single instance of the teachers name

Enviado desde mi iPhone

**Comment by samjett247** [comment]
@jlovoi Feel free to run this whenever you get a chance. Might be nice to get it done before job start? Just a thought. Let me know if I can help with anything :)

**Comment by samjett247** [comment]
@jlovoi Do you think you would be able to run this scraper this weekend to get the larger dataset (from further back) and the entries from Fall 2018 + Spring 2019?

**Comment by samjett247** [comment]
Addressed in #5
